.github/workflows/unit.yml

```diff
@@ -46,6 +46,7 @@ jobs:
 
     - run: yarn install --frozen-lockfile --network-timeout 1000000
     - run: yarn build-report
+    - run: yarn reset-link
```

Comment:
The pptr test will use whatever `lighthouse` package it resolves. Maybe instead of linking the root directory in unit tests, we make the pptr test a part of package test.

Reply:
Smoke tests for user flows can be run in package test as part of https://github.com/GoogleChrome/lighthouse/issues/13729